tests/ui/type-alias-impl-trait/issue-53398-cyclic-types.rs

```diff
@@ -1,10 +1,9 @@
 #![feature(type_alias_impl_trait)]
 
-//@ check-pass
-
 type Foo = impl Fn() -> Foo;
```

Comment:
:thinking: mind changing this to `//@ known-bug: #109268` and also adding the test for that issue?

Reply:
also, this test exists twice. This file and tests/ui/type-alias-impl-trait/type-alias-impl-trait-with-cycle-error.rs please remove this one :thinking:

please add the test from #109268 as `tests/ui/type-alias-impl-trait/type-alias-impl-trait-with-cycle-error-3.rs`, adjusting the names of the other two tests as well